addons/stock/models/stock_rule.py

```diff
@@ -288,6 +292,7 @@ def _get_stock_move_values(self, product_id, product_qty, product_uom, location_
             'description_picking': product_id._get_description(self.picking_type_id),
             'priority': values.get('priority', "1"),
             'delay_alert': self.delay_alert,
+            'orderpoint_id': values.get('orderpoint_id') and values.get('orderpoint_id').id,
```

Comment:
pedantic but the second get is useless
values.get('orderpoint_id') and values['orderpoint_id'].id,

Reply:
or you could use orderpoint_id ?